model-optimizer/mo/ops/slice.py

```diff
@@ -18,128 +18,125 @@
 
 import numpy as np
 
+from mo.front.common.partial_infer.utils import int64_array
 from mo.graph.graph import Node, Graph
 from mo.ops.op import Op
 
 
+class AttributedSlice(Op):
+    op = 'AttributedSlice'
+    enabled = True
+
+    def __init__(self, graph: Graph, attrs: dict):
+        super().__init__(graph, {
+            'type': None,
+            'op': __class__.op,
+            'in_ports_count': 1,
+            'out_ports_count': 1,
+            'infer': None,
+        }, attrs)
+
+    def supported_attrs(self):
+        return ['axis', 'start', 'end']
+
+
+class CaffeSlice(Op):
+    op = 'CaffeSlice'
+    enabled = True
+
+    def __init__(self, graph: Graph, attrs: dict):
+        super().__init__(graph, {
+            'type': None,
+            'op': __class__.op,
+            'in_ports_count': 1,
+            'out_ports_count': 1,
+            'infer': None,
+        }, attrs)
+
+    def supported_attrs(self):
+        return ['slice_point', 'axis']
+
+class TFSlice(Op):
+    op = 'TFSlice'
```

Comment:
Please, add comments for each operation describing how it works and difference between each other.

Reply:
Added informative description for each operation. Please review.